docs/images/Kiosk_Architecture.xml

```diff
@@ -1 +1 @@
EIhhHyfRz3hragJ9RuwU+++hsls1jfi11kvi8PeA0w081Ep+0K0zmOEvXtAs/6nEBF/LN2flYCYsPeJdZTU5Ev4DL57QcuOIR/cHZpGeZp7BMYQMOqTTzJo5H5PGiIkhTasoTg5PwlGkPbRI6fkG5jGJxPBdszO4DlEwV+mEWvfnITn48SMWVGO/CYDlrnFyXFdOzLhEbhN0wpTFmROP8LloZxh50I7tqDIGAT36K4yCLHmRV13GED3dDolwDOOtNF0i8R75v+Mi3fRdB20KOieQRl2SVjjEfpK6KzXE90JCphrDQ6ZziTBkol5fykTuJEFLV/tOcmrRfKe0mXk2jRBEphV9h/mHyJOt8qXCSesuyIssSawWjJYlXGabWRcgmo5YtJl1xMMLxDVlGWUSYDRhTacD0qmdCUj43OsyjMMxFXlgN5YxTfmBEsozMFYOisSd+/tGo0SOzirf0dEZ5pF6xGu2Mj54/RQ/Gx5cFxbRcw1bsj5CmuvkRVmsX84PQdvsjaTr30cEoPwpki9MQHtdm3w0OY8HGmD9NGeowpuOFacTBIiNssumUZq0GvpT4zpMMNHNqa+ZUzMeOVmVgyh4FekaDWWQyWeJdNZ86UZV7Tsx86Yj+mLIfV4RMKa8YciiO0EHLg+cxWYW5mQiyCUnn5dmp6CEoYfTA7p5rGKX+u2Ko5owrYdlunscUi4mTAIWiH2w/JRKEArFgdoMjxstsxp7jnkGXAZVDamNw7aGi5jiUVlxdIecPI6ibQc+lw75bQQ/YFfSwoxoD5UL2pXT/gn01xq5QFz7KHWsMB5a4LH8soNhCq6G5Pdgx2DBkpYeuqZyrs2PI6wFHiTXKJkOmtTjTXD9+jJkm/FgIPVBs2e5ysMsU16ZQAJL6DJr9YKGSY2vZQPZqs7zNAJek2YxMSRLEdZQrQ1UJ49ILf2N0wxbNv0uUQn2IQMDjOFguo7Eg12DwPzjL1lxBg1VGKKm6ic8k1yioML6OcVs5W4BFPpncVhe4j9OcjsC3M2O7qotj74cLdR4ULJG4INSlxgLQCwtKjLLRmEn8cFR+ONah+CHuRvbk9AmfTnVO6gjwew8eA7l0reU71Uf2H45nOLbj+Y5tQ9e0oIrgoOMYLqp9TNU2OtAAyLI807U9xwSO1wOz3FbYxeHOuORABXIQyD8q7rm6+coiMVTulyxmlW7BLxU2u16NcJrgDC8ZhCNhf8jdhlyvuiP3MZkvqMj8wAldcGIZwTfXemnBnu5LPY6Vmtin7NeIHD1joaeKcamK2+V43b64sCwN0PF7cdPy0mIA/b2sLTyoiNNreIiD+2nH1Pjpg/mFOLT+QYv/zSC8vwN/4Kvb++9nAwicnXmjnTrNFHfkYd4cPtUPDtei9RRlNYhGWwKhsd/VSawhzmlnYlMsWqTnxbyeO18h+prduWdpcH83Qd+HF7cDD/UtAHrVtqhDsyAoP1BSdXrEALCh7P3FEcVsPHsJFZIVdSp/5qkblXO7QpMegIZpNoCGK82radt01ikaQbCACo6KNDxg1AcQI0hIwzdynFIO1ANM14XnXo40LlN6Ek7C9jBPA3N8IIx9IIxS0X5uhOH6e0EYZeMACKN0dTVHV7k9vavb5Do7eTFXs5I9HGIhwU1EVtcRIl9G50EaQPvT9QD2EG94CWJ5AfJokyO0hVWvg1h0vD4oYnEQMmyvQizmayIWVxc1eReIxWlmBX3TQJoAiKuJpzu2gdrZ2FWP3X6DHOckWa7mLLgBTj8CHAeFH+5HgKPuIjcaj3cRZ7UcYFj1MKnVsC2WUYuP+s7xBFp1Sb2X26A/cRgxS/IrGdG/f6zwCr9RU5QnK8MgC5YZSd+yLSqV8Oe2RY5qi5rFD1WW4JZnCRoC+HzTUpNJrey1JLKlZLck9k1+N+VC1DBskTSppOGEF0ZoKhwaGXp0YeflqX3AL4pb5RgRRGo1AwK+ocnGQ+QZqIfgj34lJZLy7xDzetCwWoN0kPoXG1U+SuOkPNcA9Rido/LGtY26o7P3tuAVI2tcl1R99HJvdoeTJUkvY8oUE9zi9IEpy3NKn47KpWX500zo0/yI6Sr/Dbu0LWr7rGIfy9QZGMveg2tz9+DWXE+RqT3U+oD2mMoeInydEl1m93hTq+Gsh302y1Q9GOT2XlKkFx5qQg0AWzMqyDIcWBpZ12y4w5YqesrAYF3rtmAdljsKYUsK1d1ZNF8xhdotuKgowrbKw7qIiSWdVA9l9i1Mu+W9IXpbTNte+bZjsUKLbHvHz0qgbnw6Yzv9NAubIwjCfGxW0W9W0a2K9rVVpUWO/O3LoePbwLJFJ37ybS0txl9l7Ifx10yTBhQem/GHagzjnMznQRIOYpZbMcEnZi0nzKJ9eIM34w2QCwyRAD6YQ4C6wMsbdQiwe9DgXTuEjRUOXQKh4zW1K2GucTr1qDF8VFiDz6OSUL4D4vdVllsnndmQ337A4g+ldM1ZJCL/rWh/DyqIPJaZ66KG0NHIgV913UUV9QzS7ml5cbzzNiMpe9GHCc5W43smUx1Td2KTaznAAcOc7ydz9/wsNKCyBtqS0BZUVxKb0nq7SOvocmEnxPpt7aPZwB796y/RcCD2Jb6/vIqrddNVyRawVFuxv7oi/eT3W+dcKzbathv8WDP876XYaLOu/dxZf/3cWIrgveYyt1H16m5d5u5Y9aqvmHY3l0y3GsX68nij5B1N/aztwCZ+s2xkIJHH2VI0q4znNt87BLulXnoT3z1tLOciccQ5P/dgOb+Nkr7/RKAM62RZzrdWmbVtUXYnaWxPBLbchABr4rJe40V5jf4+9Db1pz+KO2g5G9A1l5zwpE9m+I6KqfqveX+aff/ieN9Ov0zT/67B91+uJ98eBj28VGOn2KjhmfLrNyjAtF7NdTx3d8cNTiM642wJXlPHhqL051+KGsf+d16wGrWG53BtA9jdRE/VIbuxImMFvc29Gy/2HrRZvQez6F69axRd/B8=</diagram></mxfile>
```

Comment:
I didn't see any overt changes, am I missing something?

Reply:
The image definitely looks different, there are multiple consumers now (instead of a stack) and the S3 Bucket has changed it's shape.